pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -140,4 +141,16 @@ int64_t         get_index(t_ndarray arr, ...);
 int64_t     *numpy_to_ndarray_strides(int64_t *np_strides, int type_size, int nd);
 int64_t     *numpy_to_ndarray_shape(int64_t *np_shape, int nd);
 
+/* numpy.sign for float, double and integers */
+inline double  sign(double x)
+{
+    return x ? x < 0 ? -1 : 1 : 0;
```

Comment:
Please add parentheses to help read this instruction

Reply:
fixed here https://github.com/pyccel/pyccel/pull/1169/commits/621c3e82d16a51ea2afdd94a346fccc6a92c01f8